authentication-ldap/src/main/java/com/synopsys/integration/alert/authentication/ldap/validator/LDAPConfigurationValidator.java

```diff
@@ -16,20 +16,16 @@ public class LDAPConfigurationValidator {
     public ValidationResponseModel validate(LDAPConfigModel ldapConfigModel) {
         Set<AlertFieldStatus> statuses = new HashSet<>();
 
-        if (null == ldapConfigModel.getEnabled()) {
-            statuses.add(AlertFieldStatus.error("enabled", AlertFieldStatusMessages.INVALID_OPTION));
-        }
-        if (null == ldapConfigModel.getIsManagerPasswordSet()) {
-            statuses.add(AlertFieldStatus.error("isManagerPasswordSet", AlertFieldStatusMessages.INVALID_OPTION));
-        }
         if (StringUtils.isBlank(ldapConfigModel.getServerName())) {
             statuses.add(AlertFieldStatus.error("serverName", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
         }
         if (StringUtils.isBlank(ldapConfigModel.getManagerDn())) {
             statuses.add(AlertFieldStatus.error("managerDn", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
         }
-        if (ldapConfigModel.getManagerPassword().isEmpty() && !Boolean.TRUE.equals(ldapConfigModel.getIsManagerPasswordSet())) {
-            statuses.add(AlertFieldStatus.error("managerPassword", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
+        if (ldapConfigModel.getManagerPassword().isEmpty()) {
+            if (Boolean.FALSE.equals(ldapConfigModel.getIsManagerPasswordSet())) {
+                statuses.add(AlertFieldStatus.error("managerPassword", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
+            }
         }
```

Comment:
```suggestion
      if (ldapConfigModel.getManagerPassword().isEmpty()
            && Boolean.FALSE.equals(ldapConfigModel.getIsManagerPasswordSet())) {
                statuses.add(AlertFieldStatus.error("managerPassword", AlertFieldStatusMessages.REQUIRED_FIELD_MISSING));
        }
```
Is this more readable while still maintaining the constructor argument you provided.